HLTrigger/Configuration/python/deepTauAtHLT.py

```diff
@@ -0,0 +1,357 @@
+                maxDeltaZ = cms.double(0.4),
+                maxTrackChi2 = cms.double(100.0),
+                maxTransverseImpactParameter = cms.double(0.1),
+                minGammaEt = cms.double(1.0),
+                minNeutralHadronEt = cms.double(30.0),
+                minTrackHits = cms.uint32(3),
+                minTrackPixelHits = cms.uint32(0),
+                minTrackPt = cms.double(0.5),
+                minTrackVertexWeight = cms.double(-1.0)
+            ),
+            vertexTrackFiltering = cms.bool(False),
+            vxAssocQualityCuts = cms.PSet(
+                maxTrackChi2 = cms.double(100.0),
+                maxTransverseImpactParameter = cms.double(0.1),
+                minGammaEt = cms.double(1.0),
+                minTrackHits = cms.uint32(3),
+                minTrackPixelHits = cms.uint32(0),
+                minTrackPt = cms.double(0.5),
+                minTrackVertexWeight = cms.double(-1.0)
+            )
+        )
+    )
+
+    process.hpsPFTauSecondaryVertexProducer = PFTauSecondaryVertexProducer.clone(
+        PFTauTag = cms.InputTag("hltHpsPFTauProducerReg")
+    )
+    process.hpsPFTauTransverseImpactParameters = PFTauTransverseImpactParameters.clone(
+        PFTauTag = cms.InputTag("hltHpsPFTauProducerReg"),
+        PFTauPVATag = cms.InputTag("hpsPFTauPrimaryVertexProducer"),
+        PFTauSVATag = cms.InputTag("hpsPFTauSecondaryVertexProducer"),
+        useFullCalculation = cms.bool(True)
+    )
+
+
+    #Add all sums and corrections for deepTauProducer
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.hpsPFTauPrimaryVertexProducer)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.hpsPFTauSecondaryVertexProducer)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.hpsPFTauTransverseImpactParameters)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.chargedIsoPtSum)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.chargedIsoPtSumdR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSum)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSumdR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.puCorrPtSum)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.footprintCorrection)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.footprintCorrectiondR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSumWeight)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.neutralIsoPtSumWeightdR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.photonPtSumOutsideSignalCone)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.photonPtSumOutsideSignalConedR03)
+    process.HLTHPSMediumChargedIsoPFTauSequenceReg.insert(-1, process.hltFixedGridRhoFastjetAll)
```

Comment:
I think it would be nicer to do this in a loop (using getattr such that you only need to write a list of the module names). However, if you used the container format resulting in less lines here, it might not be worth it. So don't take this as a strong request.